components/camel-exec/src/main/java/org/apache/camel/component/exec/ExecEndpoint.java

```diff
@@ -55,6 +56,8 @@
     private ExecBinding binding;
     @UriParam
     private boolean useStderrOnEmptyStdout;
+    @UriParam
```

Comment:
Add (default = "xxx") so we have that in the metadata

Reply:
Thanks for showing it to me, I'll fix both cases